hackerank/js-practice/camelCase.js

```diff
@@ -0,0 +1,11 @@
+function camelcase(s) {
+   let count = 1;
+   for(let i in s){
```

Comment:
Even though it's completely understandable, it would be better to use more descriptive variables